pkg/filesystem/fs/fs.go

```diff
@@ -62,9 +62,15 @@ var nativeEndian binary.ByteOrder
 type Mode int
 
 const (
+	// A single nydusd serves all container images
 	SharedInstance Mode = iota
+	// One container image one nydusd topology
 	MultiInstance
+	// Rafs is integrated with other custom components.
+	// Nydusd is not needed to work as FUSE server or other backend service.
+	// Snapshotter just cooperates with containerd with mount slice
 	NoneInstance
+	// Nydusd does not fulfill any concrete image service but prefetch data layers into local cache
```

Comment:
Nydusd does not fulfill any lazy pull (on-daemon) ability but only prefetch image data into local blob cache for other components.

Reply:
fixed